birdhouse/optional-components/stac-data-proxy/config/magpie/permissions.cfg.template

```diff
@@ -0,0 +1,12 @@
+# NOTE:
+# Assume 'secure-data-proxy' would exist if needed.
```

Comment:
I do not enforce it because `stac-data-proxy` can be used in 2 modes: 
- fully-open (similar to wps-outputs), in which case `secure-data-proxy` is irrelevant (since not enabled)
- secured access, when `secure-data-proxy` is enabled, and therefore created by that component

Reply:
All documented here: 
https://github.com/bird-house/birdhouse-deploy/blob/cc8dc1f7c1e62a8b8d7892f1c1714363fd002cd3/birdhouse/optional-components/README.rst#populate-stac-catalog-with-sample-data

Important: look at the file in raw mode, GitHub somehow does not render the `list-table` RST directive :(

Reply:
> Should I place this config under birdhouse/optional-components/stac-data-proxy/config/secure-data-proxy instead of birdhouse/optional-components/stac-data-proxy/config/magpie to make it effective only when both are combined? secure-data-proxy already depends on magpie

Yes I think that makes more sense